app/controllers/api/functions.php

```diff
@@ -160,6 +162,8 @@
     ->param('templateOwner', '', new Text(128, 0), 'The name of the owner of the template.', true)
     ->param('templateRootDirectory', '', new Text(128, 0), 'Path to function code in the template repo.', true)
     ->param('templateBranch', '', new Text(128, 0), 'Production branch for the repo linked to the function template.', true)
+    ->param('memory', 512, new Memory(), 'Memory in MB allocated for the function.', true)
+    ->param('cpus', 1, new Cpus(), 'CPU cores allocated for the function.', true)
```

Comment:
as we discussed we should replace these two params with a `size` parameter which should accept constants like `s-1vcpu-512mb` etc.